mobzy-nms-injection/src/main/kotlin/com/mineinabyss/mobzy/injection/types/HostileMob.kt

```diff
@@ -25,7 +27,8 @@ open class HostileMob(
         addPathfinderGoal(7, LookAtPlayerBehavior(radius = 8.0f))
         addPathfinderGoal(8, RandomLookAroundBehavior())
 
-        addTargetSelector(2, TargetNearbyPlayer())
+        addTargetSelector(2, TargetAttacker())
```

Comment:
Don't add new behavior here, we actually want to remove these entirely and only define in config

Reply:
wdym? Like add the serialized behavior tag to all of the hostile mobs ("mobzy:target.nearby_player")? It'd be better to have all of the hostile mobs derive behavior from the same base class and if I'm not mistaken, the base class is here, not in the server configs

Reply:
think the idea is that the base class has no pathfinders defined and all are on each mob. making it so "flying" can easily contain different types of flying mobs.
Seeing as rohana and splitjaw both use the flying base class iguess

Reply:
So like the current categorization of mobs is something that needs to be rewritten anyways? Something like Flying, Land, Water mobs with Hostile, Neutral, Passive behaviors is a target scheme?

Reply:
not entirely sure what the final idea was, as yes we could then in theory ditch HostileMob for LandMob and the rest is defined in the targetselectors and pathfindergoals @0ffz

Reply:
For now I think remaining it as default is fine, then we move it all over later at some point.

Reply:
Yeah this is *fine* for now but in the future all these base classes will be completely empty and you'll be able to inherit default behaviour using other ECS entities (so we'd have an entity for flying mobs in config)